### PR TITLE
Changing from MathJax to KaTeX

### PR DESCRIPTION
**Pull request by kerimoyle**
This PR needs https://github.com/hsorby/vue-sphinx-xml/pull/19 to go through, have another update, and then the version number here to be updated too ...

### CODE REVIEW

**Comment by hsorby** [comment]
I don't think this PR should be merged it has file level references to packages. And Katex is added as a dependency through vue-sphinx-xml.  This may not be the way to go but for now we will see how it works out.

**Comment by kerimoyle** [comment]
>> has file level references to packages

I don't understand what you mean by that?

**Comment by hsorby** [comment]
"vue-sphinx-xml": "file:../vue-sphinx-xml"

**Comment by kerimoyle** [comment]
Whoops!

**Comment by kerimoyle** [comment]
Needs to wait for https://github.com/hsorby/vue-sphinx-xml/pull/21 to be merged so that we can get a new release number, and update this PR before merging.

File: src/css/general.css [hunk 1]
```diff
@@ -291,7 +291,26 @@ th {
 }
 
 /* Formatting for KaTeX math blocks */
-div.equation {
+.equation-block {
```
Comment (hsorby) [inline src/css/general.css]: Can you clarify for me that this shouldn't be katex-block from the vue-shpinx-xml package MathBlock?

Comment (kerimoyle) [reply src/css/general.css]: It should ... and it's been changed.  You're looking at an old commit?

**Pull request merged**